README.md

```diff
@@ -419,6 +419,29 @@ ui5: {
 }
 ```
 
+### fileExport
+Type: `boolean` or `object`  
+Default: `false`
+
+Configures whether report files provided by tools like the UI5 Support Assistant are exported to the file system.  
```

Comment:
should we maybe also document how to provide those files? (window._$files array structure)

Reply:
Done